nni/compression/pytorch/utils/counter.py

```diff
@@ -26,110 +225,113 @@ def count_flops_params(model: nn.Module, input_size, custom_ops=None, verbose=Tr
     ---------
     model : nn.Module
         target model.
-    input_size: list, tuple
-        the input shape of data
+    x: tuple or tensor
+        the input shape of data or a tensor as input data
     custom_ops: dict
         a mapping of (module: custom operation)
```

Comment:
it is not clear what is the type of "custom operation", it is a function? the key is module type?

Reply:
yes, user should manually define a function for counting flops of the module